tools/rc/test/basic_test.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
```

Comment:
nit: can the file be called something more descriptive? e.g., basic_deterministic_test.sh ?